src/lmql/runtime/openai_secret.py

```diff
@@ -1,51 +1,87 @@
+from typing import Tuple
 import os
 
-openai_secret = None
-openai_org = None
-
 # get project root
 ROOT_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
 
-def get_openai_secret():
-    if "LMQL_BROWSER" in os.environ:
-        # get openai secret from JS context
-        import js
-        openai_secret = str(js.get_openai_secret())
-        openai_org = str(js.get_openai_organization())
-        
-        return openai_secret, openai_org
-    elif "LMQL_OPENAI_SECRET" in os.environ and "LMQL_OPENAI_ORG" in os.environ:
-        return os.environ["LMQL_OPENAI_SECRET"], os.environ["LMQL_OPENAI_ORG"]
-    elif "OPENAI_API_KEY" in os.environ:
-        return os.environ["OPENAI_API_KEY"], ""
-    elif any(k.startswith("OPENAI_API_KEY") for k in os.environ):
-        return os.environ.get("OPENAI_API_KEY", None), ""
```

Comment:
This close was probably(?) never working, since `os.environ.get("OPENAI_API_KEY", None)` gets exact match while `k.startswith("OPENAI_API_KEY")` is looking for strings that start with this string